CodeofConduct.md

```diff
@@ -0,0 +1,72 @@
+
+## Our Pledge 🤝
+
+We as members, contributors, and leaders pledge to make participation in the ServiceNow Hacktoberfest community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Our Standards ⚖️
+
+Examples of behavior that contributes to a positive environment for our community include:
+
+- 🌼 Using welcoming and inclusive language
+- 🤝 Being respectful of differing viewpoints and experiences
+- 🎯 Gracefully accepting constructive criticism
+- 🌍 Focusing on what is best for the community
+- 💖 Showing empathy towards other community members
+- 🤗 Demonstrating empathy and kindness toward other people
+- 📝 Giving and gracefully accepting constructive feedback
+- 🙏 Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+
+Examples of unacceptable behavior include:
+
+- 🚫 The use of sexualized language or imagery, and unwelcome sexual attention or advances
+- 🗯️ Trolling, insulting, derogatory comments, and personal or political attacks
+- 🚷 Public or private harassment
+- 🔒 Publishing others' private information without their explicit permission
+- ⚠️ Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities 🛡️
+
+Project maintainers are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that is deemed inappropriate, threatening, offensive, or harmful.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.
+
+## Scope 🌐
+
+This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the ServiceNow Hacktoberfest community in public spaces. Examples of representing our community include using an official project email address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement 🚨
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [earl.duque@servicenow.com](mailto:earl.duque@servicenow.com). All complaints will be reviewed and investigated promptly and fairly.
+
+All project team members are obligated to respect the privacy and security of the reporter of any incident.
+
+## Enforcement Guidelines 📜
```

Comment:
While there are benefits to having specific consequences outlined, is this really something we can adhere to?

Reply:
I agree. Perhaps we remove this section for now until we are able to specify consequences, or simply highlight that maintainers will be accountable to each other to make fair and reasonable adjudications.

Reply:
removed all the enumerations and just made it say:
Project maintainers will follow internally established guidelines in determining the consequences for any action they deem in violation of this Code of Conduct. These guidelines are established between the wiki of this repository and via internally agreed upon decisions within the sndevs#hacktoberfest-review channel. Actions that may be taken range from warnings to permanent exclusion from current and future events.